lua/nvim-treesitter/parsers.lua

```diff
@@ -528,9 +528,22 @@ list.hcl = {
 list.markdown = {
   install_info = {
     url = "https://github.com/MDeiml/tree-sitter-markdown",
-    files = { "src/parser.c", "src/scanner.cc" },
-    branch = "main",
+    include_dir = "tree-sitter-markdown/src",
+    files = { "tree-sitter-markdown/src/parser.c", "tree-sitter-markdown/src/scanner.cc" },
```

Comment:
`v`, `tsx`, `typescript`, `ocaml` `ocaml_interface` solve this using the `location` property. Can you either port the old ones to the `include_dir` or use `location`?

Reply:
(with preference for the latter, if feasible ;))